metpy/xarray.py

```diff
@@ -104,6 +108,39 @@ def coordinates_identical(self, other):
         # Otherwise, they match.
         return True
 
+    @property
+    def as_seconds(self):
+        """Return the data as unix timestamp (for easier time derivatives)."""
+        attrs = {key: self._data_array.attrs[key] for key in
+                 {'standard_name', 'long_name', 'axis'} & set(self._data_array.attrs)}
+        attrs['units'] = 'seconds'
+        return xr.DataArray(self._data_array.values.astype('datetime64[s]').astype('int'),
+                            name=self._data_array.name,
+                            coords=self._data_array.coords,
+                            dims=self._data_array.dims,
+                            attrs=attrs)
+
+    def axis_name(self, axis):
```

Comment:
`axis_from_name`?

Reply:
No. :sheep:

`find_axis_name`? I think I want a verb in there some how, since as a noun it's really more of a property, which isn't the case.